InferenceSystem/src/model_retrain_negative_samples.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import datetime
+import globals
+
+
+def main():
+    """
+    Example usage:
+    python3 model_retrain_negative_samples.py --start_time "2020-07-25 19:15:00" --end_time "2020-07-25 19:45:00" --hydrophone "bush_point"
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--start_time",
+        type=str,
+        required=True,
+        help="Start time in PST in following format 2020-07-25 19:15:00",
+    )
+    parser.add_argument(
+        "--end_time",
+        type=str,
+        required=True,
+        help="End time in PST in following format 2020-07-25 20:15:00. Cannot be greater than (start_time + 30 minutes)",
+    )
```

Comment:
```suggestion
    td = globals.MAX_NEGATIVE_SAMPLE_RETRAIN_DATETIME_DELTA
    max_timedelta_string = (
        f"{td.seconds // 3600}H:{(td.seconds // 60) % 60}M{td.seconds % 60}S"
    )
    parser.add_argument(
        "--end_time",
        type=str,
        required=True,
        help=f"End time in PST in following format 2020-07-25 20:15:00. Cannot be greater than (start_time + {max_timedelta_string})",
    )
```
Might want to get the minutes from the global instead of hardcoding